python/ray/data/_internal/execution/resource_manager.py

```diff
@@ -195,7 +184,7 @@ def get_op_usage_str(self, op: PhysicalOperator) -> str:
                 f"op_outputs: {memory_string(self._mem_op_outputs[op])})"
             )
             if (
-                self.op_resource_allocator_enabled()
+                isinstance(self._op_resource_allocator, ReservationOpResourceAllocator)
```

Comment:
What's the motivation behind this change?

Reply:
oh, I was cleaning up the type hint errors in this file, and noticed this.
If not changing this, mypy will complain on the next line.